Mathlib/Tactic/Attr/Register.lean

```diff
@@ -81,5 +81,5 @@ discharge theorems about the trivial case (where we know `Subsingleton α` and m
 in e.g. groups are trivially true). -/
 register_simp_attr nontriviality
 
-/-- A stub attribute for `is_poly`. -/
+/-- A stub attribute for `IsPoly`. -/
```

Comment:
This is a mis-match; shouldn't this attribute be renamed instead?
I don't see this attribute actually used in practice; according to comments, it seems unused. I'm fine reverting this, but wanted to call attention to this one.

Reply:
It should be named however is needed to get the syntax for the attribute to be in snake case. Looking at the source for `register_label_attr`, it seems that `is_poly` is correct here.